File: Código/Tareas/tarea6-intro-alopez/Figura/src/Figura.java
public class Figura {

    protected int x;
    protected int y;

    public Figura(int x, int y)
    {
        this.x=x;
        this.y=y;
    }

    public void setX(int x)
    {  if(x>=0 ^ x<=1023){
        this.x=x;}
    else{
        System.out.println("El valor de x debe ser mayor o igual a 0 y menor a 1024.");}
    }
    public int getX()
    {
        return x;
    }
    public void setY(int y)
    {  if(y>=0 ^ y<=768){
        this.y=y;}
        else{
        System.out.println("El valor de y debe ser mayor o igual a 0 y menor a 768.");}
    }
    public int getY()
    {
        return y;
    }



}

File: Código/Programación orientada a objetos/Python/Figura.py
class Fig:

    def __init__(self,x, y):

        self.__x = x

        self.__y = y

    def getX(self):

        return self.__x

    def setX(self, X):

        if X >=  0 and X <= 1023:

            self.__x = x

        else: print("El valor de x debe ser mayor o igual a 0 y menor a 1024.")

    def getY(self):

        return self.__y

    def setY(self, y):

        if y >= 0 and y<= 767:

            self.__y = y

        else: print("El valor de y debe ser mayor o igual a 0 y menor a 768.")
